cli/src/base/AccountsCommandBase.ts

```diff
@@ -82,10 +82,12 @@ export default abstract class AccountsCommandBase extends ApiCommandBase {
     const [fromBalances, toBalances] = await this.getApi().getAccountsBalancesInfo([fromAddress, toAddress])
 
     // payer account balance should be greater than amount, otherwise transfer will fail
-    checkBalance(fromBalances, new BN(amount))
+    if (fromBalances.votingBalance.lt(new BN(amount))) {
```

Comment:
- `votingBalance` should be `>= amount + existentialDeposit`
- `availableBalance` should be `>= amount`
(this ignores transaction fees however)
```suggestion
    if (
      fromBalances.votingBalance.lt(new BN(amount).add(this.getApi().existentialDeposit()))
      || fromBalances.availableBalance.lt(new BN(amount))
    ) {
```